bucket/cura.json

```diff
@@ -2,36 +2,32 @@
     "version": "4.2.1",
     "license": "LGPL-3.0-only",
     "homepage": "https://ultimaker.com/en/products/ultimaker-cura-software",
-    "description": "Ultimaker Cura prepares your model for 3D printing.",
-    "bin": "cura.exe",
+    "description": "Model editing tools for 3D printing",
     "architecture": {
         "64bit": {
-            "url": "https://github.com/Ultimaker/Cura/releases/download/4.2.1/Ultimaker_Cura-4.2.1-win64.exe",
+            "url": "https://github.com/Ultimaker/Cura/releases/download/4.2.1/Ultimaker_Cura-4.2.1-win64.exe#/dl.7z",
             "hash": "b463873df49434f2dbc960746a92a0767fe0477a80bf86903624f4b51cf4e711"
         }
     },
+    "suggest": "extras/vcredist2015",
+    "bin": "curacli.exe",
     "shortcuts": [
         [
             "cura.exe",
             "Cura"
+        ],
+        [
+            "curacli.exe",
+            "Cura (command-line interface)"
```

Comment:
What? Command line interface as shortcut?

Does this work as powershell?

Reply:
Removing command-line interface now. It seems not necessary.

Do you mean it runs properly under Powershell by "work as powershell"?

Reply:
No. I mean if that works like powershell.

When you open it from start menu it will spawn custom terminal, which handle all inputs.

Reply:
No. It runs under regular terminal.